docs/faq.rst

```diff
@@ -306,6 +306,32 @@ Run::
 Now all available conda environments (for example, one created by following the instructions `here <quickstart.rst#installation-using-conda>`_) will automatically be added to the list of kernels.
 
 
+Addressing SerializationError
```

Comment:
Maybe this text is user guide material not FAQ material

Reply:
We already have a little note here https://parsl.readthedocs.io/en/latest/userguide/apps.html#limitations. Most users don't run into this issue, so I didn't think it made sense to duplicate this content over on the limitations section. I think it makes it easier to search `SerializationError` and have a note available that explains it that we can point to from the exception message.

Reply:
Those two sections should at least cross-reference each other then, so that someone reading one of them gets to also read the other.